library_schemas/slam/prerelease/hedtsv/HED_slam_1.0.0/HED_slam_1.0.0_Tag.tsv

```diff
@@ -1,2 +1,13 @@
 hedId	rdfs:label	Level	omn:SubClassOf	Attributes	dc:description	omn:EquivalentTo	Annotations
-	Anatomical-landmark	0	Anatomical-item	rooted=Anatomical-item	A collection of specific points on the human body used for anatomical and sensor placement reference.		dc:source "Original", rdfs:comment "Body landmarks are used to standardize sensor placement."
+	Anatomical-landmark	0	Anatomical-item	rooted=Anatomical-item	A collection of specific points on the human body used for anatomical and sensor placement reference.		dc:source "Original", rdfs:comment "Body landmarks are used to standardize sensor placement.
+	Head-landmark	1	Anatomical-landmarks		Points on the head used as reference landmarks.		dc:source "Original"
+	Nasion	2	Head-landmark		The midpoint between the eyes just above the bridge of the nose.		
+	Inion	2	Head-landmark		The most prominent point of the occipital bone at the back of the head.		
+	Left Helix-Tragus Junction (LHJ)	2	Head-landmark		The point where the helix meets the tragus of the left ear.		
+	Right Helix-Tragus Junction (RHJ)	2	Head-landmark		The point where the helix meets the tragus of the right ear.		
+	Vertex	2	Head-landmark		The highest point on the head in the midline.		
+	Mastoid-process	2	Head-landmark				dc:source "Original"
+	Torso-chest-landmark	1	Anatomical-landmarks		Points on the chest used as reference landmarks.		dc:source "Original"
+	Acromion-process	2	Torso-chest-landmark		The outermost point of the shoulder blade.		has_dbxref: "NCIT:C32048" "FMA:12525" "UBERON:0002497"
```

Comment:
The syntax here is incorrect.  Assuming that we were using obo: as the namespace prefix and `has_dbxref` as the AnnotationProperty, the syntax would be:

obo:has_dbxref  "NCIT:C32048",  obo:has_dbxref "FMA:12525", obo:has_dbxref  "UBERON:0002497"

However, we need to do some research on exactly what AnnotationProperty we want to use --- this is one that everyone will want to use.